sample-plugin/pom.xml

```diff
@@ -103,6 +103,12 @@
             <artifactId>workflow-job</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Not sure if I need to update the tests to exercise this dependency. If so, is it enough to just add an unused import to verify that the dependency is available during compilation?

Reply:
Not necessary in this case. We just require that the sample plugin’s POM exercises all dependencies to verify that it can be done without triggering Enforcer violations or the like.